README.rst

```diff
@@ -38,26 +39,34 @@ Operating FFTW in multithreaded mode is supported.
 pyFFTW implements the numpy and scipy fft interfaces in order for users to
 take advantage of the speed of FFTW with minimal code modifications.
 
-A comprehensive unittest suite can be found with the source on the github
-repository or with the source distribution on PyPI.
+A comprehensive unittest suite can be found with the source on the GitHub_
+repository or with the source distribution on PyPI_.
 
-The documentation can be found on
-`Read the Docs <http://pyfftw.readthedocs.io>`_, the source is
-on `github <https://github.com/pyFFTW/pyFFTW>`_ and the python package
-index page is `here <http://pypi.python.org/pypi/pyFFTW>`_.
+The documentation can be found on `Read the Docs`_, the source is on GitHub_
+and the python package index page (PyPI_).  Issues and questions can be
+raised at the `GitHub Issues`_ page.
 
 Requirements (i.e. what it was designed for)
 --------------------------------------------
-- Python 2.7 or greater (Python 3 is supported)
-- Numpy 1.6
-- FFTW 3.3 or higher (lower versions *may* work) libraries for single, double,
+- Python_ 2.7 or >= 3.4
+- Numpy_ >= 1.10.4  (lower versions *may* work)
+- FFTW_ >= 3.3 (lower versions *may* work) libraries for single, double,
   and long double precision in serial and multithreading (pthreads or openMP)
-  versions
-- Cython 0.15 or higher (though the source release on PyPI loses this
-  dependency)
+  versions.
+- Cython_ >= 0.23 (lower versions *may* work)
 
 (install these as much as possible with your preferred package manager).
 
+In practice, pyFFTW *may* work with older versions of these dependencies, but
+it is not tested against them.
+
+Optional Dependencies
+---------------------
+- Scipy_ >= 0.16
+- Dask_ >= 0.14.2
```

Comment:
Should be bumped to 0.15.0.

Reply:
Should be resolved with PR ( https://github.com/grlee77/pyFFTW/pull/2 ).

Reply:
Guess I forgot to update this in the text as well. Would you like to update or would you like me to send another PR?